pmd-apex/src/main/resources/category/apex/design.xml

```diff
@@ -408,6 +408,37 @@ public class Person {
     Date birthDate;
     BodyMeasurements measurements;
 }
+]]>
+        </example>
+    </rule>
+
+    <rule name="AvoidUnusedMethod"
+          since="7.0.0"
+          message="Unused methods make understanding code harder"
+          class="net.sourceforge.pmd.lang.apex.rule.design.AvoidUnusedMethodRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_apex_design.html#avoidunusedmethod">
+        <description>
+            Avoid having unused methods since they make understanding and maintaining code harder.
```

Comment:
If I understand this correctly, then not only private unused methods are found, but also public methods, that are not in use by any other class. That's maybe worth mentioning here in the documentation.